src/Modal/Modal.test.jsx

```diff
@@ -7,7 +7,8 @@ import Button from '../Button';
 
 
 const modalOpen = (isOpen, wrapper) => {
-  expect(wrapper.childAt(0).hasClass('modal-open')).toEqual(isOpen);
+  expect(wrapper.find('.modal').hasClass('modal-open')).toEqual(isOpen);
+  expect(wrapper.find('.modal').hasClass('modal-backdrop')).toEqual(isOpen);
```

Comment:
should there be a test for `show` and `fade` or am I missing something?

Reply:
Added :)